### PR TITLE
Elite Tactical Magboots

### PR DESCRIPTION
**Pull request by matttheficus**
## What Does This PR Do
Re-sprites the Advanced Syndicate Magboots and renames them to "Elite Tactical Magboots".
Color palette taken from the Elite Syndicate MODSuit.

Changes a few loadouts to use Elite Tactical Magboots instead of Advanced, as it looks better than the BRIGHT white booties.
- All Gamma ERT members.
- Deathsquad
- SolGov LT
- SolGov Elite/Elite LT
- SST
- Syndicate Officers

## Why It's Good For The Game
This continues the Syndicate Item (though in this case, since the only way to get these is via "strike team" style roles, its going to fall under more than just syndicate items) trend of naming the "advanced" versions "Elite".

Allows allows a more tacticool coloring for SST/Syndie Officer/SolGov magboots.

## Images of changes
image

## Testing
Put on the new drip.

## Changelog
:cl:
tweak: Advanced Blood Red Magboots are now Elite Tactical Magboots.
imageadd: added sprites for the Elite Tactical Magboots.
/:cl:

### CODE REVIEW

**Comment by Coolrune206** [comment]
Traitors with CE Magboot steal can no longer mug the singular Gamma ERT Janitor I send to clean up the station.
It's so over.

File: code/modules/clothing/shoes/magboots.dm [hunk 1]
```diff
@@ -78,10 +78,14 @@
 	magboot_state = "syndiemag"
 	origin_tech = "magnets=4;syndicate=2"
 
-/obj/item/clothing/shoes/magboots/syndie/advance //For the Syndicate Strike Team
-	name = "advanced blood-red magboots"
-	desc = "Reverse-engineered magboots that appear to be based on an advanced model, as they have a lighter magnetic pull. Property of Gorlex Marauders."
+/obj/item/clothing/shoes/magboots/elite //For the Syndicate Strike Team/SolGov/Tactical Teams
+	name = "elite tactical magboots"
+	desc = "Advanced magboots used by strike teams across the System. Allows for tactical insertion into low-gravity areas of operation."
```
Comment (Burzah) [inline code/modules/clothing/shoes/magboots.dm:83]:
```suggestion
	desc = "Advanced magboots used by strike teams across the system. Allows for tactical insertion into low-gravity areas of operation."
```
System is not a proper noun, so it should not be capitalized in this instance.

**Review by Burzah — COMMENTED** [review]
Code looks good, but there is just one small nit pick to look over.

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]
deconflict for merge

**Comment by matttheficus** [comment]
> deconflict for merge

Done

**Pull request merged**